cms/src/App.vue

```diff
@@ -71,6 +72,17 @@ onBeforeMount(async () => {
 
             await loginRedirect();
         });
+
+        socket.on("changeRequestAck", (data: any) => {
+            if (data.ack == "rejected") {
+                useNotificationStore().addNotification({
+                    title: `Changed request: ${data.ack}`,
+                    description: data.message,
+                    state: "error",
+                    timer: 4000,
```

Comment:
In error cases I think the error needs to stick around for a long time to give the user time to see it and take action (e.g. take a screenshot). I would set it to at least 1 minute.

Reply:
Added